autokeras/auto_model.py

```diff
@@ -364,6 +378,23 @@ def _prepare_data(self, x, y, validation_data, validation_split):
                 validation_split)
         return dataset, validation_data
 
+    def _get_x(self, dataset):
```

Comment:
This method would benefit form a docstring, since the name is not self-explanatory